internal/service/users_v1/ocm_interface.go

```diff
@@ -0,0 +1,47 @@
+package users
+
+import (
+	"context"
+	"os"
+
+	sdk "github.com/openshift-online/ocm-sdk-go"
+	"github.com/openshift-online/ocm-sdk-go/logging"
+)
+
+type OcmSDK struct{}
+
+func (ocm *OcmSDK) InitSdkConnection(ctx context.Context) (*sdk.Connection, error) {
+	// Create a logger that has the debug level enabled:
+	logger, err := logging.NewGoLoggerBuilder().
```

Comment:
Oh - I was hoping we could also maybe shim out our logger to satisfy this interface too. but we can do that in a followup later.